ConnectSDKTests/Services/Helpers/WebOSTVServiceSocketClientTests.m

```diff
@@ -95,4 +95,58 @@ - (void)testDeniedPairingShouldCallRegistrationFailed {
                                  }];
 }
 
+-(void)testPairingShouldCallSocketWillregister{
```

Comment:
Please use the proper capitalization: `testPairingShouldCallSocketWillRegister.`